From 677ed1de9faea3ad1b312e3ffa6ce3ffd6471d53 Mon Sep 17 00:00:00 2001
From: Andrei Botila <andrei.botila@nxp.com>
Date: Tue, 8 Feb 2022 12:11:10 +0200
Subject: [PATCH 18/43] Revert "s32g399ardb3: enable default ATF boot flow"

This reverts commit d6f39cb23d9f8a447cebb44c1d8aaaf954ddd8e9.
The S32_ATF_BOOT_FLOW has been removed from code. Since the removal of
STANDALONE support the S32_ATF_BOOT_FLOW is no longer necessary.

Issue: ALB-8347
Upstream-Status: Pending 

Signed-off-by: Andrei Botila <andrei.botila@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 configs/s32g399ardb3_defconfig      | 1 -
 configs/s32g399ardb3_qspi_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configs/s32g399ardb3_defconfig b/configs/s32g399ardb3_defconfig
index 43c32857aa..6dac51e558 100644
--- a/configs/s32g399ardb3_defconfig
+++ b/configs/s32g399ardb3_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G399ARDB3=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g399ardb3_qspi_defconfig b/configs/s32g399ardb3_qspi_defconfig
index 4f577c93cb..29cca72956 100644
--- a/configs/s32g399ardb3_qspi_defconfig
+++ b/configs/s32g399ardb3_qspi_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G399ARDB3=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
-- 
2.17.1

